package-lock.json

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
We have this under `./render`. Let's remove this file and package.json from this PR.

Reply:
Done